{ 'high_step': { 'left_arm': [ { 'duration': 2.952,
                                 'name': u'high_step_5',
                                 'positions': [ -1.6698,
                                                -0.0332,
                                                -0.6098,
                                                0.0544,
                                                -0.0012,
                                                0.0001,
                                                0.0],
                                 'starttime': 0.0}],
                 'left_leg': [ { 'duration': 3.016,
                                 'name': u'high_step_0',
                                 'positions': [ 0.0099,
                                                -0.0536,
                                                0.0021,
                                                -0.0014,
                                                -0.0063,
                                                -0.0546],
                                 'starttime': 0.0},
                               { 'duration': 4.024,
                                 'name': u'high_step_0_left',
                                 'positions': [ -0.0038,
                                                -0.1993,
                                                0.0007,
                                                -0.0005,
                                                0.0027,
                                                -0.1973],
                                 'starttime': 3.016},
                               { 'duration': 2.932,
                                 'name': u'high_step_1_left',
                                 'positions': [ 0.0008,
                                                -0.1963,
                                                -0.2525,
                                                0.5059,
                                                0.2557,
                                                -0.2007],
                                 'starttime': 7.04},
                               { 'duration': 4.192,
                                 'name': u'high_step_2_left',
                                 'positions': [ 0.01,
                                                -0.3279,
                                                -1.4502,
                                                2.4264,
                                                0.9951,
                                                -0.22],
                                 'starttime': 9.972},
                               { 'duration': 2.83,
                                 'name': u'high_step_3_left',
                                 'positions': [ 0.0102,
                                                -0.3275,
                                                -1.517,
                                                1.5889,
                                                0.1983,
                                                -0.2201],
                                 'starttime': 14.164},
                               { 'duration': 2.952,
                                 'name': u'high_step_4_left',
                                 'positions': [ 0.01,
                                                -0.1648,
                                                -1.52,
                                                1.59,
                                                0.2,
                                                -0.0629],
                                 'starttime': 16.994},
                               { 'duration': 3.145000000000003,
                                 'name': u'high_step_4_1_left',
                                 'positions': [ 0.01,
                                                -0.1649,
                                                -1.4576,
                                                1.5538,
                                                0.1635,
                                                -0.0629],
                                 'starttime': 19.946},
                               { 'duration': 3.147,
                                 'name': u'high_step_5_left',
                                 'positions': [ 0.01,
                                                -0.0392,
                                                -1.4572,
                                                1.8067,
                                                0.528,
                                                0.0156],
                                 'starttime': 23.091000000000005}],
                 'neck': [],
                 'right_arm': [ { 'duration': 2.952,
                                  'name': u'high_step_5',
                                  'positions': [ 1.6698,
                                                 0.0332,
                                                 0.6098,
                                                 -0.0544,
                                                 0.0012,
                                                 -0.0001,
                                                 -0.0],
                                  'starttime': 0.0}],
                 'right_leg': [ { 'duration': 3.016,
                                  'name': u'high_step_0',
                                  'positions': [ -0.0099,
                                                 0.0536,
                                                 -0.0021,
                                                 0.0014,
                                                 0.0063,
                                                 0.0546],
                                  'starttime': 0.0},
                                { 'duration': 4.002,
                                  'name': u'high_step_0_right',
                                  'positions': [ 0.0225,
                                                 -0.1666,
                                                 -0.0149,
                                                 0.0022,
                                                 0.0094,
                                                 -0.1544],
                                  'starttime': 3.016},
                                { 'duration': 3.145,
                                  'name': u'high_step_5_right',
                                  'positions': [ 0.02,
                                                 -0.0437,
                                                 -0.1025,
                                                 -0.0467,
                                                 -0.2793,
                                                 0.0478],
                                  'starttime': 23.14}],
                 'torso': [ { 'duration': 3.03,
                              'name': u'high_step_0',
                              'positions': [-0.0, 0.0],
                              'starttime': 0.0}]}}
